prog/vm/nexus.rb

```diff
@@ -258,8 +268,21 @@ def before_run
     when "Succeeded"
       host.sshable.cmd("common/bin/daemonizer --clean prep_#{q_vm}")
       vm.private_subnets.each(&:incr_add_new_nic)
+      # To simulate failure case in development:
+      # if frame["attempt"] < 3
+      #   Clog.emit("VM prep forced fail for #{vm.ubid}, attempt: #{frame["attempt"]}")
```

Comment:
Clog is really only useful in its structured form for log analysis, if doing this, maybe do `puts`.

Reply:
I'll include a change to this comment in another PR.